components/event-publisher-proxy/pkg/metrics/collector.go

```diff
@@ -9,21 +9,21 @@ import (
 
 const (
 	// Errors name of the errors metric
-	Errors = "event_publish_to_messaging_server_errors_total"
+	Errors = "eventing_epp_errors_total"
 	// Latency name of the latency metric
-	Latency = "event_publish_to_messaging_server_latency"
+	Latency = "eventing_epp_messaging_server_latency_duration_nanoseconds"
```

Comment:
we record the latency in milliseconds! should we rename this to milliseconds instead of nanoseconds?

Reply:
Done!